client/stats/host.go

```diff
@@ -80,6 +83,8 @@ func (h *HostStatsCollector) Collect() (*HostStats, error) {
 			Free:      memStats.Free,
 		}
 		hs.Memory = ms
+	} else {
```

Comment:
I agree that removing `if/else` statements. But `return nil, err` is OK?

~~~
if err != nil {
   return nil, err
}
~~~

I thought it should collect errors as below like you commented.

~~~
mErr.Errors = append(mErr.Errors, fmt.Errorf("error to collect cpu stats", err))
~~~